java-checks-test-sources/src/main/java/checks/aws/AwsLongTermAccessKeysCheck.java

```diff
@@ -0,0 +1,18 @@
+package checks.aws;
+
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.auth.BasicSessionCredentials;
+
+public class AwsLongTermAccessKeysCheck {
+  void noncompliant() {
+    BasicAWSCredentials foo = new BasicAWSCredentials("", ""); // Noncompliant [[sc=35;ec=54]] {Make sure using a Long-term access key is safe here.}
```

Comment:
The issue message should be checked by putting the expected string between double accolades.
```suggestion
    BasicAWSCredentials foo = new BasicAWSCredentials("", ""); // Noncompliant [[sc=35;ec=54]] {{Make sure using a Long-term access key is safe here.}}
```